scene/3d/skeleton_3d.cpp

```diff
@@ -265,11 +265,28 @@ void Skeleton3D::_update_process_order() {
 
 	bones_backup.resize(bones.size());
 
+	_update_bone_names();
+
```

Comment:
I think this should set `concatenated_bone_names = StringName()` and only update in the getter.